2-mandatory/2-financial-times.js

```diff
@@ -6,6 +6,15 @@
 */
 function potentialHeadlines(allArticleTitles) {
     // TODO
+    const newArray = [];
+    for (let articleTitle of allArticleTitles){
+        if (articleTitle.length <= 65){
+            newArray.push(articleTitle)
+        }
+    }
+    return newArray;
```

Comment:
Good! Although it's good to try and make variable names reflect what they are doing. For example here, a better name than `newArray` would have been, `potentialHeadlines` or similar

Reply:
This is another good function Melese. Easy for someone else to follow.